### DISCUSSION TITLE
invalid decimal literal

### DISCUSSION THREAD

**Discussion by brainb203** [rytilahti/python-miio#1421] [Q&A]
![螢幕擷取畫面 2022-05-18 154304](https://user-images.githubusercontent.com/104008223/168985057-f11d5ce6-8fc1-4872-9e78-ea70deb175b9.png)

invalid decimal literal

any one can help?

**Comment by rytilahti** [comment]
Looks like you have one too many `"` in your payload for the `ssid` key. Try `'{"ssid": "", "pswd": "123123 ;  ..."}`
